.travis.yml

```diff
@@ -2,28 +2,45 @@ matrix:
   include:
 
     - name: "Python 3.6 on Linux"
-      python: "3.6"
       os: linux
       dist: xenial
       language: python
+      python: 3.6
+      service:
+        - xvfb
+      script:
+        - xvfb-run -a python3 setup.py test
```

Comment:
Noob question here: is the xvfb mandatory here since we don't need a X11 server running (except for jobs with optional dependencies)?

Reply:
Yes, unfortunately — the reason is the `TkinterScheduler` tests, which are always run (because tkinter is distributed with python itself).

Reply:
Thanks!